docs/source/getting_started/faq.rst

```diff
@@ -0,0 +1,25 @@
+Frequently Asked Questions (FAQ)
+================================
+
+Below are some frequently asked questions regarding the beta release
+series. Click on a question to be directed to relevant information in
+our documentation or our GitHub repo.
+
+General
+-------
+
+-  :ref:`What is JupyterLab? <overview>`
+-  :ref:`How stable is JupyterLab? <stability>`
+-  :ref:`What will happen to the classic Jupyter Notebook? <classic>`
+-  :ref:`How long will JupyterLab be in beta? <beta>`
+-  `Where is the official online documentation for
```

Comment:
Do we want this link to point somewhere else now that they're actually in the docs eg the latest docs or the API?